scripts/generatePagesFromConsolidatedJSON.py

```diff
@@ -442,7 +442,7 @@ def generateAllPages(pathPages,paper):
  */
  (function() { // DON'T EDIT BELOW THIS LINE
  var d = document, s = d.createElement('script');
- s.src = 'https://replicability-siggraph.disqus.com/embed.js';
+ s.src = 'https://replicability.graphics.disqus.com/embed.js';
```

Comment:
```suggestion
 s.src = """ +  """'https://replicability-siggraph.disqus.com/embed.js'; """ if variant['DOI']=="10.1145/3197517.3201378" else """'https://replicability-siggraph.disqus.com/embed.js'""" +
"""
```

Reply:
completely untested ;)

Reply:
crap.. one should be "." and not "-"

Reply:
i've just learned inline if

Reply:
Twice `replicability-siggraph` ?
And right it should be a `.`

Reply:
/shrug

Reply:
I've fixed it, are you fine with the suggested change ?

Reply:
have you commit/pushed  something?

Reply:
I don't see any edits (beside my crappy suggestion)

Reply:
I just edited your suggestion, if we agree on it we can commit directly.

Reply:
woh I didn't know that we (and especially you) could edit a suggestion...